lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -1440,6 +1451,30 @@ module Resources
         'op': 'remove',
         'path': '/relationship/0'
       },
+      {
+        'op': 'replace',
```

Comment:
you can't remove a relationship and then replace the extension on it.

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/662/commits/64dc7efa0c6226c1d502e96383687635615a478f#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR1474-R1477

Reply:
Added a test operation with index 1 here https://github.com/cerner/fhir.cerner.com/pull/662/commits/68a2ee4cc8f0b46eee616422c60b847998c06caa#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR1474-R1478